README.md

```diff
@@ -8,7 +8,7 @@ omnisharp-roslyn
 
 OmniSharp-Roslyn is a .NET development platform based on [Roslyn](https://github.com/dotnet/roslyn) workspaces. It provides project dependencies and language syntax to various IDE and plugins.
 
-OmniSharp-Roslyn is built with the [.NET Core SDK](https://dot.net/) on Windows and [Mono](http://www.mono-project.com/) on OSX/Linux. It targets the __net46__ target framework. OmniSharp requires __mono__ (>=5.2.0) if it is run on a platform other than Windows.
+OmniSharp-Roslyn is built with the [.NET Core SDK](https://dot.net/) on Windows and [Mono](http://www.mono-project.com/) on OSX/Linux. It targets the __net461__ target framework. OmniSharp requires __mono__ (>=5.2.0) if it is run on a platform other than Windows.
```

Comment:
Are some of the new dependencies `netstandard` only (>1.3), or what was the reason for targeting `net461`?

Reply:
This is to allow for netstandard2.0 loading and support.  In addition https://github.com/OmniSharp/csharp-language-server-protocol is now netstandard2.0 (which is what caused me to start this separately)